### DISCUSSION TITLE
[Feature Request]: Select Icon Set/Customize icons to use

### DISCUSSION THREAD

**Discussion by orjanalmen** [rappasoft/laravel-livewire-tables#1485] [Feature Requests]
### Overview

It would be great to be able to use the same icon set as everything else on the site we are building. 

### Detailed explanation

Maybe some way of setting in the config file for icon set or which icon to use where.

**Comment by lrljoe** [comment]
I like the idea!  What I think will probably end up being the end solution is allowing people to specify each icon, rather than a set 

As all of the sets use different naming conventions, it wouldn't just be a case or
"use x iconset".

There's a few bits on the cards at the moment, but I've added this to the project.

If you're keen/eager for this feature, then feel free to open a PR that offers this, so long as the standard behaviour is a default, there's plenty of approaches, feel free to reach out on discord if you want clarity on what would be expected for an easy merge into the core code.

**Comment by lrljoe** [comment]
@orjanalmen - I've set this to Low Priority as there are other feature requests that I think will be more widely used.

We have only recently migrated away from using a ton of SVGs embedded.

If you have a pressing need for this, then a PR would 100% be welcome from yourself.  
Alternatively, you should be able to achieve this (at your own risk), by publishing the views. Noting that these are subject to change at short notice, especially while this is in beta.

Ultimately I would imagine that we'll use some placeholders, and allow customisation via config files, but it's not on (at least my radar) for a month or two.
